addons/sales_team/models/crm_team_member.py

```diff
@@ -137,6 +137,13 @@ def _compute_member_warning(self):
                 else:
                     member.member_warning = False
 
+    @api.model
+    def _read_group_crm_team_ids(self, teams, domain, order):
```

Comment:
Just to keep the file organized, could you move it in the CRUD section (probably below _synchronize_memberships ) ?